demisto_sdk/commands/common/tools.py

```diff
@@ -550,3 +551,17 @@ def find_type(path: str):
                 return 'indicatorfield'
 
     return ''
+
+
+def disable_logs(name, level):
+    """
+    disable logs from channel name starting from equal and bigger than level
+
+    Arguments:
+        name -  channel name
+        level - enum for a specific level which represent the max level from which logs will be printed.
+                e.g. :Logger.ERROR = 40 - all logs that are ERROR level an grater will be printed , all lower will not
+    """
+    log = logging.getLogger(name)
+    log.level = level
+    log.manager._clear_cache()
```

Comment:
this is a bit of overkill, why did we create a function for that?

Reply:
I added name and level and made helper function ,so we will be able to disable any logger and any messages in other parts of the code if needed.

Reply:
if i understand correctly, the cache clearing is required just because we're setting minimum log level
if thats the case, i dont like the fact we're touching an internal mechanism just for that
i prefer we do it like you did in the first place, where you disable log outputs only for `pykwalify`, dont think we really need this generalization

Reply:
The problem with what I did before is that logger.disable, disabled all logger's logs pykwalify and others. What you suggested could work but the problem is that there is no disable function by levels to log = logger.getLogger() and as a result I had to set level and clean cache

Reply:
did you try `.setLevel(logging.WARNING)` ?

Reply:
SetLevel did exactly what I did in the helper function. So I changed it.

Reply:
not sure i follow.. if it does what you want, why didnt you use it in the first place and created a function?
is the code currently leads to the expected results?

Reply:
Yes the code leads to expected results, I did not found the SetLevel function and as a result I created one that does the samething